reconcile/utils/terrascript_aws_client.py

```diff
@@ -288,13 +288,41 @@ def __init__(
 
             ts += provider.template(version="2.2.0")
 
+            access_key_backend_value = config["aws_access_key_id"]
+            secret_key_backend_value = config["aws_secret_access_key"]
+            bucket_backend_value = ""
+            key_backend_value = ""
+            region_backend_value = ""
```

Comment:
you don't need to declare the variables here. in python, local variables are scoped within the function, so assigning them later in this function will make them available during Backend creation.